src/Nethermind/Nethermind.Consensus/Processing/ReadOnlyTxProcessingEnv.cs

```diff
@@ -14,27 +14,28 @@
 // ReSharper disable UnusedAutoPropertyAccessor.Global
 namespace Nethermind.Consensus.Processing
 {
-    public class ReadOnlyTxProcessingEnv : IReadOnlyTxProcessorSource
+    internal sealed class ReadOnlyTxProcessingEnv : IReadOnlyTxProcessorSource
```

Comment:
Naming is a bit confusing due to both `Source` and `Env`.

```suggestion
    internal sealed class ReadOnlyTxProcessorSource : IReadOnlyTxProcessorSource
```